drivers/sensor/lis2dh/lis2dh.c

```diff
@@ -83,7 +83,7 @@ static int lis2dh_sample_fetch(const struct device *dev,
 {
 	struct lis2dh_data *lis2dh = dev->data;
 	size_t i;
-	int status;
```

Comment:
Is there a reason you changes status to ret?  Does it serve a purpose?

Reply:
Some functions use ret, others used status. I changed all status to ret to make the code more consistent. It doesn't serve any functional purpose.